micropsi_integration_sdk/robot_sdk.py

```diff
@@ -274,6 +274,16 @@ def command_stop(self) -> None:
         """
         raise NotImplementedError
 
+    @staticmethod
+    def set_wait_for_receive_timeout() -> float:
```

Comment:
is this a setter function, or a getter?
I would rename it `get_wait_for_receive_timeout()` instead.

Reply:
It gets the value from robot interface/sdk, makes sense!